src/VirtualClient/VirtualClient.Monitors/FileUploadMonitor.cs

```diff
@@ -114,6 +114,7 @@ private async Task ProcessFileUploadsAsync(IBlobManager blobManager, EventContex
                 catch (OperationCanceledException)
                 {
                     // Expected whenever ctrl-C is used. Do a check once more, without cancellationToken and break;
+                    await this.UploadFilesAsync(blobManager, telemetryContext);
```

Comment:
This resolves an issue where the last set of logs for a given VC run were not getting uploaded on blob

Reply:
Is it possible for us to get stuck here? if the upload takes forever? We should nevertheless put a hard stop to prevent VC fails to stop when there is a long backlog of files to upload